app/script/localization/translations/webapp-cs.js

```diff
@@ -248,8 +248,6 @@ z.string.cs.conversation_context_menu_delete_everyone = 'Odstranit pro všechny'
 z.string.cs.conversation_context_menu_download = 'Stáhnout';
 z.string.cs.conversation_context_menu_like = 'Líbí';
 z.string.cs.conversation_context_menu_unlike = 'Nelíbí';
-z.string.cs.conversation_delete_timestamp = 'Smazáno v {{date}}';
-z.string.cs.conversation_edit_timestamp = 'Upraveno v {{date}}';
```

Comment:
Do not delete these

Reply:
I actually didn't delete anything, as those are just the translations I get from crowdin.